ui/app/styles/app.css

```diff
@@ -275,6 +275,10 @@ label.required::before {
   vertical-align: middle;
 }
 
+.value {
```

Comment:
prefer using generics ( for example https://github.com/project-ncl/pnc-web-ui-angularjs/blob/master/ui/app/styles/generics.css#L94 ) in this case, `.value` class is too generic and there is high change it can cause unwanted changes

Reply:
Ok, did not notice the generics file. That's nice standard.